resources/i18n/ko.json

```
@@ -26,7 +26,7 @@
       "MyAccountInformation": "사용자 정보 변경",
       "MyAccount": "사용자 정보 변경",
       "FullName": "사용자 이름",
-      "FullNameInvalid": "사용자 이름은 64자 이하로 입력하십시오",
+      "FullNameInvalid": "사용자 이름은 0자 이상 64자 이하로 입력하십시오",
```

```suggestion
      "FullNameInvalid": "사용자 이름은 64자 이하로 입력하십시오",
```
"0자 이상" is not necessary because this field is required.